.mage/go.go

```diff
@@ -53,135 +49,3 @@ func (Go) CheckVersion() error {
 	}
 	return nil
 }
-
-var goModule = "go.thethings.network/lorawan-stack"
-
-func goBuildVersionFlags() (string, error) {
-	commit, branch, tag, err := git.Info()
-	if err != nil {
-		return "", err
-	}
-	var flags []string
-	for k, v := range map[string]string{
-		"GitCommit": commit,
-		"GitBranch": branch,
-		"TTN":       tag,
-		"BuildDate": now.Format("2006-01-02T15:04:05Z"),
-	} {
-		flags = append(flags, fmt.Sprintf("-X %s/pkg/version.%s=%s", goModule, k, v))
```

Comment:
How do we insert the build/version info now?

Reply:
See https://goreleaser.com/build/
The default `ldflags` are `-s -w -X main.version={{.Version}} -X main.commit={{.ShortCommit}} -X main.date={{.Date}}`
Isn't that enough?(also in response to https://github.com/TheThingsNetwork/lorawan-stack/pull/109#pullrequestreview-201993397)

Reply:
OK with me, but then `main` needs to set those in `pkg/version`, as those are the ones that are actually used by the stack.

Reply:
This is what we get now(https://github.com/rvolosatovs/lorawan-stack/releases/tag/v3.0.2):
```
docker run --rm rvolosatovs/lorawan-stack:3.0.2 cli version
The Things Network Stack for LoRaWAN: ttn-lw-cli
Version:             3.0.2
Go version:          go1.11.5
OS/Arch:             linux/amd64
```